services/app-api/handlers/reports/fetch.ts

```diff
@@ -49,7 +53,7 @@ export const fetchReport = handler(async (event, _context) => {
       Key: `${buckets.FORM_TEMPLATE}/${state}/${formTemplateId}.json`,
     };
 
-    const formTemplate = await s3Lib.get(formTemplateParams); // TODO: strict typing
+    const formTemplate = (await s3Lib.get(formTemplateParams)) as AnyObject; // TODO: strict typing
```

Comment:
Comment: `formTemplate` type is [this one](https://github.com/Enterprise-CMCS/macpro-mdct-mcr/blob/e12110fa9ab9e2ad9f66d87e15b532816949c46e/services/ui-src/src/types/index.ts#L38-L47).

Reply:
ty, we will copy that over into app-api.

Reply:
Sure thing